### PR TITLE
[Theme] Noctis

### PR DESCRIPTION
**Pull request by 0rphee**
A port of the [Noctis](https://github.com/liviuschera/noctis#noctis) theme. 
noctis

### CODE REVIEW

File: runtime/themes/noctis.toml [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+'ui.selection.primary' = { bg = "mid-green" } # The primary selection when there are multiple.
+'ui.cursorline.primary' = { bg = 'mid-green' }
+
+'ui.linenr' = { bg = "dark-green", fg = "gray"  } # Line numbers.
+'ui.linenr.selected' = { bg = "mid-green", fg = "light-green", modifiers = [ "bold" ] } # Current line number.
+
+'ui.virtual' = { fg = "mid-green" } # Namespace for additions to the editing area.
+'ui.virtual.ruler' = { bg = "mid-green" } # Vertical rulers (colored columns in editing area).
+'ui.virtual.whitespace' = { fg = "light-gray"} # Whitespace markers in editing area.
+# NOTE: Indent guides aren't yet added at the time of writing, but will be soon.
```
Comment (the-mikedavis) [inline runtime/themes/noctis.toml:43]: actually indent guides are now merged 🎉 #1796

File: runtime/themes/noctis.toml [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+
+'ui.popup' = { bg = "mid-green" } # Documentation popups (space-k).
+# 'ui.ppopup.info' = { bg = "midgreen", fg = "gray", modifiers = ["bold"] } # Info popups box (space mode menu).
+
+# SYNTAX HIGHLIGHTING ==============================
+# All the keys here are Treesitter scopes.
+
+'property' = { fg = "red" } # Regex group names.
+'special' = { fg = "mid-blue"} # Special symbols e.g `?` in Rust, `...` in Hare.
+'module' = { } # Unclear, used for Heex module.
```
Comment (the-mikedavis) [inline runtime/themes/noctis.toml:64]: The module key should now be gone: https://github.com/helix-editor/helix/commit/e4e8a39bf79734e70248303782d9b97ef4b3540d

Comment (0rphee) [reply runtime/themes/noctis.toml]: changed now!

**Review by the-mikedavis — APPROVED** [review]
This looks very nice, thanks! 🎨

**Comment by the-mikedavis** [comment]
Could you add the screenshot to the themes wiki? https://github.com/helix-editor/helix/wiki/Themes

**Comment by 0rphee** [comment]
Sure, done!

**Pull request merged**